recipes/sofa/recipe.yaml

```diff
@@ -0,0 +1,185 @@
+          then: make
+        - if: win
+          then: ninja
+      host:
+        - libboost-headers
+        - zlib
+        - tinyxml2
+        - eigen
+        - cxxopts
+        - gtest
+      run:
+        - zlib
+      run_exports:
+        - ${{ pin_subpackage('libsofa', upper_bound='x.x.x') }}
+    tests:
+      - package_contents:
+          lib:
+            - Sofa.Core*
+    about:
+      summary: Runtime librairies for the SOFA framework
+      description: Runtime librairies for SOFA, a real-time multi-physics
+       simulator with an emphasis on medical simulation and robotics.
+      license: LGPL-2.1-or-later
+      license_file:
+        - LICENSE-LGPL.md
+        - extlibs/difflib/LICENSE.MIT
+        - extlibs/json/LICENSE.MIT
+        - extlibs/stb/LICENSE.MIT
+
+  - package:
+      name: sofa-devel
+    build:
+      script: scripts/build-devel
+      files:
+      # exclude everything that has been packaged in libsofa subpackage
+        exclude:
+          - ${{ "Library/" if win }}lib/${{ "lib" if not win }}Sofa*
+          - ${{ "Library/" if win }}plugins/**/lib
+          - if: win
+            then:
+              - Library/bin/*
+              - Library/plugins/**/bin
+        include:
+          - ${{ "Library/" if win }}include/*
+          - ${{ "Library/" if win }}lib/cmake/Sofa*
+          - ${{ "Library/" if win }}plugins/**/include
+          - ${{ "Library/" if win }}plugins/**/lib/cmake
+    requirements:
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - if: unix
+          then: make
+        - if: win
+          then: ninja
+      host:
+        - libboost-headers
+        - zlib
+        - tinyxml2
+        - eigen
+        - cxxopts
+        - gtest
+      run:
+        - ${{ pin_subpackage('libsofa', exact=True) }}
+      run_exports:
+        - ${{ pin_subpackage('libsofa', upper_bound='x.x.x') }}
+    tests:
+      - package_contents:
+          include:
+            - Sofa.Core/sofa/core/init.h
+          files:
+            - ${{ "Library/" if win }}lib/cmake/Sofa.Core/Sofa.CoreConfig.cmake
+            - ${{ "Library/" if win }}lib/cmake/Sofa.Core/Sofa.CoreConfigVersion.cmake
+    about:
+      summary: Development librairies for the SOFA framework
+      description: Development librairies for SOFA, a real-time multi-physics
+       simulator with an emphasis on medical simulation and robotics.
+      license: LGPL-2.1-or-later
+      license_file:
+        - LICENSE-LGPL.md
+        - extlibs/difflib/LICENSE.MIT
+        - extlibs/json/LICENSE.MIT
+        - extlibs/stb/LICENSE.MIT
+
+  - package:
+      name: sofa-gl
+    build:
+      script: scripts/build-sofa-gl
+    requirements:
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib('c') }}
+        - cmake
+        - if: unix
+          then: make
+        - if: win 
+          then: ninja
+        - if: linux
+          then: ${{ cdt('mesa-libgl-devel') }}
```

Comment:
The use of CDT for GL packages is now superseded, you can just depend on `libgl-devel` as a normal host package, and everything should work fine.

Reply:
Oh I've misunderstood that, thanks, that is now clear. Fixed.